collectors/ebpf.plugin/README.md

```diff
@@ -198,6 +198,7 @@ You can configure the information shown on `outbound` and `inbound` charts with
 ```conf
 [network viewer]
     maximum dimensions = 500
+    resolve hostname ips = no
     ports = 1-1024 !145 !domain
     hostnames = !example.com
```

Comment:
i assume there is implicit _deny all_ at the end, it means `hostnames = !example.com` == deny all.

is it so?

Reply:
If you are denying only `example.com`, I think we should allow all the domains that does not have `example.com`.  Why would we deny everything when only `example.com` was denied?
I understand deny all as `hostnames = !*`

Reply:
because it is common pattern - **imlicit deny** if no match. If you want 

> allow all the domains that does not have

this should be explicitly written as

```
hostnames = !example.com *
```

---

Simple patterns follows same logic (implicit deny). All filters i saw follow it (ACL, permits lists on network devices, etc).

Reply:
`ips`/`ports`/`hostnames` filters should work like that. If you have doubts for some reason asky any other person.

Reply:
I understood your point of view, I was trying to simplify the user's life only.
I will change the current behavior in few hours.

Reply:
Hi @ilyam8 ,

I was reviewing the code now and I observed that we are doing exactly what you said, but I confused with the default behavior. When nobody defines `ports` and `hostnames`, we do not reject by default, we accept all these values.

Reply:
because default is

```ini
[network viewers]
  ports = *
  ips = *
```

or whatever default we want

Reply:
exactly, this was always the case.
I will mark this as resolved, because we reached an agreement.